### PR TITLE
Bump version to 1.0

### PR DESCRIPTION
**Pull request by dcharkes**

### CODE REVIEW

File: pubspec.yaml [hunk 1]
```diff
@@ -1,13 +1,11 @@
...
 homepage: https://github.com/dart-lang/ffi
 description: Utilities for working with Foreign Function Interface (FFI) code.
 
 environment:
   sdk: '>=2.12.0-259.9.beta <3.0.0'
 
-# dependencies:
-
 dev_dependencies:
   pedantic: ^1.10.0-nullsafety
```
Comment (mit-mit) [inline pubspec.yaml:10]: Can we roll these deps?

https://pub.dev/packages/pedantic/versions/1.10.0
https://pub.dev/packages/test/versions/1.16.0

File: pubspec.yaml [hunk 2]
```diff
@@ -1,13 +1,11 @@
 name: ffi
-version: 0.3.0-nullsafety.2
+version: 1.0.0-nullsafety.0
```
Comment (mit-mit) [inline pubspec.yaml:2]: We'll need an actual 1.0.0 stable release soon also. Is this 1.0.0 pre-release to ensure things are working first?

File: pubspec.yaml (hunk 1 above)
Comment (dcharkes) [reply pubspec.yaml]: done

File: pubspec.yaml (hunk 2 above)
Comment (dcharkes) [reply pubspec.yaml]: Done

**Review by mit-mit — APPROVED** [review]

**Comment by kevmoo** [comment]
@lrhn ?

**Comment by dcharkes** [comment]
> @lrhn ?

We don't need a reply from @lrhn here, that was for https://github.com/dart-lang/ffi/pull/83.

I'm running a final g3 sanity check (cl/352762630) before merging this.

**Comment by kevmoo** [comment]
Woo hoo!

**Pull request merged**